go.mod

```diff
@@ -16,18 +16,24 @@ require (
 	cosmossdk.io/store v1.1.1
 	cosmossdk.io/x/feegrant v0.1.1
 	cosmossdk.io/x/upgrade v0.1.4
+	github.com/99designs/keyring v1.2.2
 	github.com/BurntSushi/toml v1.4.0
+	github.com/ChainSafe/go-schnorrkel/1 v0.0.0-00010101000000-000000000000
 	github.com/CosmWasm/wasmd v0.55.0-beta.0
+	github.com/StirlingMarketingGroup/go-namecase v1.0.0
 	github.com/atotto/clipboard v0.1.4
 	github.com/avast/retry-go/v4 v4.5.1
 	github.com/cometbft/cometbft v0.38.15
 	github.com/cosmos/cosmos-sdk v0.50.11
+	github.com/cosmos/go-bip39 v1.0.0
 	github.com/cosmos/gogoproto v1.7.0
 	github.com/cosmos/ibc-go/modules/capability v1.0.1
 	github.com/cosmos/ibc-go/v10 v10.0.0-beta.0.0.20250218182351-fcb6809d58dd
+	github.com/cosmos/ibc-go/v8 v8.0.0-20230906115913-46ee5f92e1af
```

Comment:
Does this work? From what I recall if you import both v8 and v10 you get some global error registry failure when you run them. This was the reason we ended up forking to remove polkadot, penumbra, etc. that were not upgraded to v10 yet.